cogs/zalgo.py

```diff
@@ -43,5 +58,10 @@ def zalgoify(self, text, amount=3):
         return zalgo_text
 
 def setup(bot):
-    n = Zalgo(bot)
+    '''set up'''
+    n = Zalgo(bot) # pylint: disable=invalid-name
     bot.add_cog(n)
+    # fekkin pylint

```

Comment:
I don't mind if you put it in a comment of your PR, but please don't include this, and the below as a comment here.